lib/evss/disability_compensation_form/service_exception.rb

```diff
@@ -35,6 +35,14 @@ class ServiceException < EVSS::ServiceException
         PIFInUse: 'evss.disability_compensation_form.pif_in_use'
       }.freeze
 
+      # Retry if any upstream external service unavailability exceptions (unless it is caused by an invalid EP code)
+      # and any PIF-in-use exceptions are encountered.
+
+      def retryable?
+        (@key == 'evss.external_service_unavailable' && only_has_retriable_message_texts?) ||
+          (@key == 'evss.disability_compensation_form.pif_in_use')
```

Comment:
```suggestion
        (@key == ERROR_MAP[:serviceError] && only_has_retriable_message_texts?) ||
          (@key == ERROR_MAP[: PIFInUse])
```

Reply:
OTOH, it's odd that `ERROR_MAP` contains multiple keys with the same value - `serviceError` & `ServiceException`. Not sure which is appropriate to use.